website/blog/2023-02-24-Deploy-databend-on-minio.md

```diff
@@ -0,0 +1,211 @@
+
+- Storage Layer
+
+The Databend storage layer is the Databend Fuse Engine and is persisted using object storage in the cloud or self-built object storage. Databend uses the Parquet format on data block storage and implements min/max indexing, sparse indexing, bloom indexing, etc.
+
+
+### Databend support deployment environments
+
+| Environments| Databend |
+| ----------- | -------- |
+| AWS S3      | Yes     |
+| Google GCS  | Yes     |
+| Azure Blob  | Yes     |
+| Aliyun OSS  | Yes    |
+| Tencent COS | Yes     |
+| Huawei OBS  | Yes     |
+| MinIO       | Yes     |
+| Ceph        | Yes     |
+| Wasabi      | Yes     |
+| SeaweedFS   | Yes     |
+| QingStor    | Yes     |
+
+Detailed reference: https://databend.rs/doc/deploy/deploying-databend
+
+## Databend Single-machine deployment
+
+> In essence, the minio environment is relatively easy to set up. However, if you use in production, you can use the MinIO Cloud or AWS S3 environment to reduce hassle.
+
+The following uses the MinIO + Databend standalone deployment in Linux of x64 as an example:
+
+| Software | path           | port                                                |
+| -------- | -------------- | --------------------------------------------------- |
+| minio    | /data/minio    | 9900                                                |
+| databend | /data/databend | mysql: 3307 <br/> http:  8000 <br/>Clickhouse http:  8124 |
+
+### Minio Deployment
+
+MinIO Homepage: https://min.io/
+
+Download MinIO from the official website and start by running the following commands:
+
+```Bash
+cd /data 
+mkdir minio
+cd minio 
+wget https://dl.min.io/server/minio/release/linux-amd64/minio
+export MINIO_ROOT_USER=minioadmin
+export MINIO_ROOT_PASSWORD=minioadmin
+./minio server ./data./minio server --address :9900 ./data
+```
+Use the MinIO Admin web interface to create a "databend" bucket.
+
+![](../static/img/blog/databend-minio-beginner-01-2.png)
+
+After creating the bucket, the MinIO deployment is complete.
+
+### Download and Install Databend
+
+| type                                                         | os            |
+| ------------------------------------------------------------ | ------------- |
+| - databend-version-nightly-x86_64-unknown-linux-musl.tar.gz  | Linux for x64 |
+| - databend-version-nightly-aarch64-apple-darwin.tar.gz       | MacOS of M1   |
+| - databend-version-nightly-aarch64-unknown-linux-musl.tar.gz | Linux of Arm  |
+| - databend-version-nightly-x86_64-apple-darwin.tar.gz        | MacOS of X64  |
+| - source code                                                | src           |
+
+We can download the musl package for Linux from the Databend open-source project repo a: https://github.com/datafuselabs/databend/tags
```

Comment:
We can download the musl package for Linux from the Databend open-source project repo: https://github.com/datafuselabs/databend/tags